axelrod/strategies/prober.py

```diff
@@ -102,6 +102,74 @@ def strategy(self, opponent):
                 return D if opponent.history[-1:] == [D] else C
 
 
+class Prober4(Player):
+    """
+    Plays C, C, D, C, D, D, D, C, C, D, C, D, C, C, D, C, D, D, C, D initially.
+    Counts retaliating and provocative defections of the opponent.
+    If the absolute difference between the counts is smaller or equal to 2,
+    defects forever.
+    Otherwise plays C for the next 5 turns and TFT for the rest of the game.
+
+    Names:
+
+    - prober4: [PRISON1998]_
+    """
+
+    name = 'Prober 4'
+    classifier = {
+        'stochastic': False,
+        'memory_depth': 1,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    @init_args
+    def __init__(self):
+        Player.__init__(self)
+        self.init_sequence = [
+            C, C, D, C, D, D, D, C, C, D, C, D, C, C, D, C, D, D, C, D
+        ]
+        self.just_Ds = 0
```

Comment:
This is a meta comment for everyone (@drvinceknight @meatballs): we could consider counting CC, CD, DC, DD in the player class like we do for cooperations and defections, since multiple players make use of it.